modules/text_server_adv/text_server_adv.cpp

```diff
@@ -1728,7 +1728,10 @@ bool TextServerAdvanced::shaped_text_update_breaks(RID p_shaped) {
 			if (is_whitespace(c)) {
 				sd_glyphs[i].flags |= GRAPHEME_IS_SPACE;
 			}
-			if (u_ispunct(c)) {
+			if (c == 0x005F) {
+				sd_glyphs[i].flags |= GRAPHEME_IS_UNDERSCORE;
```

Comment:
nitpicking but seeing the other checks, should we have an is_underscore(c) function here that checks this? Probably overkill but I can see there may be a reason alternatives to underscores are added at some point.

Reply:
It seems there is one in `text_server_fb` so it should likely be added here too for consistency.

Reply:
ah, thanks, I didn't see that there was a method for this check